### PR TITLE
Added __debugInfo()

### PR DESCRIPTION
**Pull request by robertpustulka**
See #129

### CODE REVIEW

**Comment by lorenzo** [comment]
Would you mind adding a couple simple tests? Only to prevent bad method calls in the future

File: src/Chronos.php [hunk 1]
```diff
@@ -127,4 +127,20 @@ public function copy()
...
+
+    /**
+     * Return properties for debugging.
+     *
+     * @return array
+     */
+    public function __debugInfo()
+    {
+        $properties = [
+            'currentTime' => $this->format('Y-m-d H:i:s.u'),
```
Comment (markstory) [inline src/Chronos.php]: currentTime implies 'now' and not this object's time.

Comment (dereuromark) [inline src/Chronos.php]: Would `value` work?

Comment (robertpustulka) [inline src/Chronos.php]: Maybe just `time` and `date`?

Comment (inoas) [inline src/Chronos.php]: `date`, `time`, `timezone`?

Comment (markstory) [inline src/Chronos.php]: time and date work for me.

**Comment by lorenzo** [comment]
Thanks!

File: src/MutableDateTime.php [hunk 2]
```diff
@@ -175,7 +175,7 @@ public function __set($name, $value)
     public function __debugInfo()
     {
         $properties = [
-            'currentTime' => $this->format('Y-m-d H:i:s.u'),
+            'time' => $this->format('Y-m-d H:i:s.u'),
```
Comment (dereuromark) [inline src/MutableDateTime.php]: dateTime

**Pull request merged**